content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,498 @@
+
+## Requests
+
+All the API endpoints below can have two different host endpoints:
+
+* If you are on the Datadog US site: `https://api.datadoghq.com/api/`
+* If you are on the Datadog EU site: `https://api.datadoghq.eu/api/`
+
+### Create a new Authentication mapping
+
+
+Create a new AuthN Mapping from a JSON body. Returns the newly created AuthN Mapping.
+
+| Method | Endpoint path | Required payload |
+|--------|--------------|------------------|
+| `POST`  | `/v2/authn_mappings`  | JSON       |
+
+##### ARGUMENTS
+
+* **`id`** [*required*, *default*=none]:
+  The UUID of the `authinmap` being updated. For more information about finding your UUID, see the [Role API documentation][1].
+* **`role_uuid`** [*required*, *default*=none]:
+  The Roles API can be used to create and manage Datadog roles, what global permissions they grant, and which users belong to them. When you create a Role, it is assigned a UUID. For more information about finding the `role_uuid` for the role you are updating, see the [Role API documentation][1].
+* **`attribute_key`** [*required*, *default*=none]:
+ The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+* **`attribute_value`** [*required*, *default*=none]:
+  The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X POST \
+         "https://app.datadoghq.com/api/v2/authn_mappings/{UUID}" \
+         -H "Content-Type: application/json" \
+         -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+         -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+         -d '{
+             "data": {
+                 "type": "authn_mappings",
+                 "id": "123e4567-e89b-12d3-a456-426655440000",
+                 "attributes": {
+                      "role_uuid": "123e4567-e89b-12d3-a456-426655445555",
+                      "attribute_key": "string",
+                      "attribute_value": "string"
+                }
+             }
+         }'
+```
+{{< /code-block >}}
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholder with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+{{< code-block lang="json" filename="response.json" disable_copy="true" >}}
+{
+  "data": {
+    "attributes": {
+      "created_at": "timestamp",
+      "created_by": "string",
```

Comment:
Not your fault but this is about to change again actually 😞. The `created_by` field used to be user id's (which were integers), but to match the other new rbac api's this will become a `uuid`. So string isn't technically wrong, but probably best to do like we've done with the other example uuid's and put something that matches those :)

Reply:
Actually, on second thought, might have to get back to you on this one. Another option would be to remove the field altogether, since it isn't entirely necessary to be there on every request. Will consult with the team to see what the best option here is going forward.